config/database.yml

```diff
@@ -58,7 +58,7 @@ development:
 test:
   <<: *default
   database: WarAndPiece_test
-
+  template: template0
```

Comment:
You probably don't want to commit these changes. The `database.yml` file shouldn't really need to change.

Reply:
How do I make this change consistent on my local machine without pushing it? Almost everything I did would gave me errors before I made this change